static/app/actionCreators/modal.tsx

```diff
@@ -250,7 +250,7 @@ export async function openInviteMembersModal({
 
 type InviteMissingMembersModalOptions = {
   allowedRoles: OrgRole[];
-  missingMembers: {integration: string; users: MissingMember[]};
+  missingMembers: MissingMember[];
```

Comment:
i realized since this is only for github we can vastly simplify everything by not having the integration name here for now